components/Navbar.tsx

```diff
@@ -1,62 +1,147 @@
 "use client";
 
+import { useState, useEffect, useRef } from "react";
 import Image from "next/image";
 import Link from "next/link";
-import { DropdownMenu } from "./DropdownMenu";
-import { motion } from "framer-motion";
+import { motion, AnimatePresence } from "framer-motion";
+import { Menu, X } from "lucide-react";
 
 export default function Navbar() {
+  const [isOpen, setIsOpen] = useState(false);
+  const navRef = useRef<HTMLElement>(null);
+
+  const toggleMenu = () => setIsOpen(!isOpen);
+
+  useEffect(() => {
+    const handleClickOutside = (event: MouseEvent) => {
+      if (navRef.current && !navRef.current.contains(event.target as Node)) {
+        setIsOpen(false);
+      }
+    };
+
+    document.addEventListener("mousedown", handleClickOutside);
+    return () => {
+      document.removeEventListener("mousedown", handleClickOutside);
+    };
+  }, []);
+
   return (
-    <header className="container text-gray-100 mx-auto px-4 py-6 flex items-center justify-between">
-      <motion.div
-        initial={{ opacity: 0, x: -20 }}
-        animate={{ opacity: 1, x: 0 }}
-        transition={{ duration: 0.5 }}
-      >
-        <Link href="/" className="flex items-center justify-center gap-4">
-          <Image
-            src="/logos/appkom-logo.svg"
-            alt="Appkom logo"
-            width={50}
-            height={50}
-            className="h-10 w-10"
-          />
-          <span className="text-xl font-bold">Appkom</span>
-        </Link>
-      </motion.div>
-
-      <motion.nav
-        className="hidden md:flex items-center gap-8"
-        initial={{ opacity: 0, y: -10 }}
-        animate={{ opacity: 1, y: 0 }}
-        transition={{ duration: 0.5, delay: 0.2 }}
-      >
-        <Link
-          href="/prosjekt"
-          className="hover:text-onlineOrange transition-colors"
-        >
-          Prosjekter
-        </Link>
-        <Link
-          href="/blogg"
-          className="hover:text-onlineOrange transition-colors"
-        >
-          Blogg
-        </Link>
-        <Link
-          href="/medlem"
-          className="hover:text-onlineOrange transition-colors"
-        >
-          Medlemmer
-        </Link>
-        <Link
-          href="/kontakt"
-          className="px-4 py-2 bg-onlineBlue text-white rounded-full hover:bg-opacity-70 transition-colors"
-        >
-          Ta kontakt
-        </Link>
-        <DropdownMenu /> {/* TODO: fix mobile menu */}
-      </motion.nav>
+    <header
+      ref={navRef}
+      className="fixed top-0 left-0 right-0 z-50 text-gray-100 bg-transparent/70"
```

Comment:
Hva med å gjøre bakgrunnen litt blurry?
![image](https://github.com/user-attachments/assets/098edbf6-baf1-43f5-a590-8800d0201c27)
```suggestion
      className="fixed top-0 left-0 right-0 z-50 text-gray-100 bg-transparent/70 backdrop-blur-sm"
```

Reply:
🔥🔥